alphalens/tears.py

```diff
@@ -138,9 +139,9 @@ def create_factor_tear_sheet(factor,
                                                                                1,
                                                                                std_err=std_quant_daily)
 
-    factor_autocorrelation = perf.factor_rank_autocorrelation(factor,
-                                                              time_rule='D')
-
+    factor_autocorrelation = pd.concat(
+        [perf.factor_rank_autocorrelation(factor, period=p) for p in periods], axis=1)
```

Comment:
I agree and thinking at issue #68 too, we might call the option `turnover_for_all_periods` and plot both autocorrelation and turnover plots (interleaved) for all periods when it is enabled ?